app/views/roasts/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= render "shared/partials/nav-global" %>
+<%= render "shared/partials/header-global" %>
+
+<!-- first loop around main and second loop around roasts_brands section -->
+<!-- can loop the markup through Rails since each roast is templated -->
+
+<section class="roasts__slider-wrapper">
+    <div class="slider-axis">
+      <%= image_tag("light-slider-bean.svg", alt:"") %>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <%= image_tag("dark-slider-bean.svg", alt:"") %>
+    </div>
+    <div class="slider"><input type="range" value="0" min="0" max="8" step="1" class="slider__range"></div>
+</section>
+
+<main class = "roasts">
+  <% roasts.each_with_index do |roast, index| %>
+    <% roastIndex = index %>
+    <section class = "roasts__view" id = "roast<%= index %>">
+      <div class="view__image-wrapper">
+        <%= image_tag(roast.image.thumb('200x200#').url, alt:"") %>
+      </div>
+      <h2><%= roast.name %></h2>
+      <p><strong><%= roast.description %></strong></p>
```

Comment:
It appears to be slightly more bold! There is a description and a sub_description, the latter which has more text.